openvdb/Types.h

```diff
@@ -316,6 +316,71 @@ namespace internal {
 
 // Implementation details of TypeList
 
+struct NullType {};
```

Comment:
Please add a comment describing what this "dummy" type is used for

Reply:
this comment applies to more then the struct listed above (please see review summary)